src/OpenTelemetry.Instrumentation.Dependencies/OpenTelemetry.Instrumentation.Dependencies.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net452;net461</TargetFrameworks>
```

Comment:
Just FYI the .NET Framework instrumentation is turned on via `#if NET461` directives so we'll need to update those in order to actually instrument net452. I'm thinking the reflection will still work fine without changes. The .NET version it spawned from targetted net45 so I checked reference source that far back to make my revisions. But good idea to validate it works after the target is updated.
Example:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/5da574c109d1f6787dec08c47d44d7c294c2832c/src/OpenTelemetry.Instrumentation.Dependencies/Implementation/HttpWebRequestActivitySource.net461.cs#L16

Reply:
thank you for reviewing. i will check where we have the tag net461 and add 452 and test.

Reply:
Should we rename the filename or should we create a new one for net452?

Reply:
Maybe rename to end in ".net4x" instead? @SergeyKanzhelev I think you were the one who originally asked for the extension to be used when we were targeting something for an entire file, how do you feel about that?

Reply:
we will update from net461 to netframework as cijo suggested